scripts/rollback-manual.sh

```
@@ -0,0 +1,53 @@
+    exit 1
+fi
+
+print_status "Rolling back from backup: $LATEST_BACKUP"
+
+# Stop current services
+print_status "Stopping current services..."
+ssh $SERVER "cd $REMOTE_DIR && docker-compose down --remove-orphans || true"
+
+# Restore from backup
+print_status "Restoring from backup..."
+ssh $SERVER "cd $REMOTE_DIR && 
```

Shellcheck (0.10.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)